src/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -464,8 +417,17 @@ const smartBidder: PrebidBidder = {
 // There's an IX bidder for every size that the slot can take
 const indexExchangeBidders = (
 	slotSizes: HeaderBiddingSize[],
+	isInVariant: boolean,
 ): PrebidBidder[] => {
-	const indexSiteId = getIndexSiteId();
+	let indexSiteId = getIndexSiteId();
+	if (isInVariant) {
```

Comment:
Non-blocking, but it would be nice to have a more specific parameter name than 'isInVariant' to specify exactly which variant we're talking about, and for more consistency with the rest of the code

Reply:
That's a good point. I have updated